index.html

```diff
@@ -15,14 +15,24 @@
 </head>
 
 <body>
-    <header class="header">
+    <header class="header-mobile">
         <nav class="nav">
-            <a class="brand" href="#">Welcome</a>
-            <a class="menu fa fa-bars fa-xl"></a>
+            <ul class="mobile-nav">
+                <li class="mobile-nav-item"><a class="brand" href="#">Welcome</a></li>
+                <li class="mobile-nav-item"><a class="menu fa fa-bars fa-xl"></a></li>
+            </ul>
         </nav>
     </header>
-
     <section class="headline">
+        <header class="header">
+            <nav class="nav-desk">
+                <ul class="desktop-nav">
+                    <li class="desktop-nav-item"><a href="#" class="nav-link">Portfolio</a></li>
```

Comment:
Kindly make your links for navigation to work by using '''id''' to the section on the page